json/pickler/src/test/scala/sttp/tapir/json/pickler/Fixtures.scala

```diff
@@ -79,3 +79,14 @@ object Fixtures:
   }
 
   case class StatusResponse(status: Status)
+
+  case class SealedVariantContainer(v: SealedVariant)
```

Comment:
maybe also introduce a test case with `enum`s (parameterless and not)

Reply:
(unless we already don't have one :) )

Reply:
We have such tests
1. Parameterless enums https://github.com/softwaremill/tapir/blob/3d981a7fc95a06d9f1ceca80e72a12ad9bd7ae75/json/pickler/src/test/scala/sttp/tapir/json/pickler/PicklerTest.scala#L448
2. With parameters https://github.com/softwaremill/tapir/blob/3d981a7fc95a06d9f1ceca80e72a12ad9bd7ae75/json/pickler/src/test/scala/sttp/tapir/json/pickler/PicklerTest.scala#L478

Reply:
Ah ok, sorry, I didn't unwrap enough context :) But maybe if they will be in separate files it will be easier to see what's already there

Reply:
(btw. open api docs tests are grouped in a similar way - each test has its own test classes in the companion object etc.)